src/DevYeah.LMS/DevYeah.LMS.Data/Models/Topic.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DevYeah.LMS.Data.Models
+{
+    public partial class Topic
+    {
+        public Topic()
+        {
+            Resource = new HashSet<Resource>();
```

Comment:
Interesting choice of data structure. It auto ignore duplicates, but, do we have such case?